tests/test_bonfire_lib/test_deploy.py

```
@@ -0,0 +1,335 @@
+"""Tests for bonfire_lib.deploy module."""
+
+import pytest
+from unittest.mock import MagicMock, patch, call
```

Flake8 (7.3.0)

[error] 'unittest.mock.call' imported but unused

(F401)